post-create

```diff
@@ -13,7 +13,7 @@ hook-post-create-global-cert() {
 
   if fn-is-ssl-enabled "$PLUGIN_CONFIG_ROOT"; then
     # we should export certs_set as a function in the certs plugin so that this works with the remote cli...
-    dokku certs:add "$APP" "$CRT_FILE" "$KEY_FILE"
+    dokku certs:add "$CRT_FILE" "$KEY_FILE"
```

Comment:
I think this is an error - the certs command takes an APP as the first argument.